gamestonk_terminal/reports/due_diligence.ipynb

```diff
@@ -114,7 +114,10 @@
    "source": [
     "from gamestonk_terminal.stocks.due_diligence import finviz_view\n",
     "\n",
-    "finviz_view.analyst(ticker=ticker, export='')"
+    "try:\n",
+    "    finviz_view.analyst(ticker=ticker, export='')\n",
```

Comment:
The export = '' shouldn't be needed

Reply:
I thought the same, but it failed when I tried.
<img width="559" alt="Screenshot 2021-11-01 at 01 48 49" src="https://user-images.githubusercontent.com/25267873/139610551-bcb65129-14df-4083-b037-34a3e0d5527e.png">

I'm about to do a PR to fix something else Colin reported, I'll also add an optional export="" to analyst